lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/aten_ltc_ts_type.cpp

```diff
@@ -174,6 +174,11 @@ at::Tensor LazyNativeFunctions::cat(at::TensorList tensors, int64_t dim) {
   return result;
 }
 
+at::Tensor LazyNativeFunctions::clone(const at::Tensor & self, c10::optional<at::MemoryFormat> memory_format) {
+  auto self_lt = bridge::GetLtcTensor(self);
```

Comment:
is it okay to ignore `memory_format`?

Reply:
I'm not sure. 

It's at least OK in the sense that torch/xla implements it this way.  But we may need to revisit (?) as we discuss channels last with nv folks.